graphql/resolvers/User/index.js

```diff
@@ -1,49 +1,37 @@
 import User from "../../../models/User";
+import { ObjectID } from "mongodb";
 
 export default {
   Query: {
-    user: (root, args) => {
-      return new Promise((resolve, reject) => {
-        User.findOne(args).exec((err, res) => {
-          err ? reject(err) : resolve(res);
-        });
-      });
+    login: async (root, { email, password }) => {
+      return await User.findOne({ email, password }).exec();
     },
```

Comment:
just changed all to async functions, shorter code. and if it should be fail, that it will still return error.